scripts/install

```diff
@@ -77,87 +91,61 @@ ynh_app_setting_set --app=$app --key=final_path --value=$final_path
 
 # Create a dedicated NGINX config
 ynh_add_nginx_config
 
-#=================================================
-# CREATE DEDICATED USER
-#=================================================
-ynh_script_progression --message="Configuring system user..." --weight=3
-
-# Create a system user
-ynh_system_user_create --username=$app
-
 #=================================================
 # PHP-FPM CONFIGURATION
 #=================================================
-ynh_script_progression --message="Configuring PHP-FPM..."
+ynh_script_progression --message="Configuring PHP-FPM..." --weight=2
 
 # Create a dedicated PHP-FPM config
 ynh_add_fpm_config
+phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
 
 #=================================================
 # SPECIFIC SETUP
 #=================================================
 # INITIALIZE DATABASE
 #=================================================
-ynh_script_progression --message="Configuring Baïkal..." --weight=3
+ynh_script_progression --message="Configuring $app..." --weight=3
 
-ynh_mysql_connect_as --user=$db_name --password="$db_pwd" --database=$db_name \
-    < "${final_path}/Core/Resources/Db/MySQL/db.sql"
+ynh_mysql_connect_as --user=$db_user --password="$db_pwd" --database=$db_name \
+    < "$final_path/Core/Resources/Db/MySQL/db.sql"
 
 #=================================================
 # CONFIGURE BAIKAL
 #=================================================
+ynh_script_progression --message="Adding a configuration file..." --weight=1
 
-bk_conf="${final_path}/config/baikal.yaml"
-timezone=$(cat /etc/timezone)
-password_hash=$(echo -n admin:BaikalDAV:$password | sha256sum | cut -d ' ' -f 1)
-ynh_app_setting_set --app=$app --key=password_hash --value=$password_hash
+#bk_conf="${final_path}/config/baikal.yaml"
 path=${path_url%/}
-deskey=$(ynh_string_random 24)
-ynh_app_setting_set --app=$app --key=encrypt_key --value="$deskey"
-
-ynh_add_config --template="../conf/baikal.yaml" --destination="$bk_conf"
-
-# Disable installation
-touch "${final_path}/Specific/INSTALL_DISABLED"
 
-#=================================================
-# GENERIC FINALIZATION
-#=================================================
-# SECURE FILES AND DIRECTORIES
-#=================================================
-
-# Set permissions
-chown -R root: "$final_path"
-chown $app "$final_path/config/baikal.yaml"
+ynh_add_config --template="../conf/baikal.yaml" --destination="$final_path/config/baikal.yaml"
+chown $app: "$final_path/config/baikal.yaml"
 chmod 640 "$final_path/config/baikal.yaml"
 
-#=================================================
-# SETUP SSOWAT
-#=================================================
-ynh_script_progression --message="Configuring permissions..." --weight=2
-
-# Allow public access on /
-ynh_permission_update --permission="main"  --add="visitors"
-# But restrain on /admin
-ynh_permission_create --permission="admin" --url="/admin" --allowed="all_users"
```

Comment:
Why did you remove the admin permission? (Also, you don't remove it in the update, so old baikal installations will have this permission while new ones won't)

Reply:
I was wondering why we need this permission...
I mean, isn't the `/admin` path  allowed to all users by default?

Reply:
We need this permission because baikal must be public to visitors. And we want to protect `/admin`.